app/fileManager.js

```diff
@@ -181,10 +189,17 @@ export default function(state, emitter) {
       const speed = size / (time / 1000);
       await delay(1000);
       await fadeOut('download-progress');
-      saveFile(f);
-      state.storage.totalDownloads += 1;
-      metrics.completedDownload({ size, time, speed });
-      emitter.emit('pushState', '/completed');
+      const pt = convertPlainText(f);
+      if (pt !== null) {
+        state.fileInfo.plaintext = convertPlainText(f);
```

Comment:
Should this just be `state.fileInfo.plaintext = pt;` since we've already converted to plain text above on L192?